qt/MainWindow.cc

```diff
@@ -150,9 +150,7 @@ MainWindow::MainWindow(Session& session, Prefs& prefs, TorrentModel& model, bool
     QIcon const icon_pause = icons.getThemeIcon(QStringLiteral("media-playback-pause"), QStyle::SP_MediaPause);
     QIcon const icon_open = icons.getThemeIcon(QStringLiteral("document-open"), QStyle::SP_DialogOpenButton);
     ui_.action_OpenFile->setIcon(icon_open);
-    ui_.action_AddURL->setIcon(
-        addEmblem(icon_open, QStringList() << QStringLiteral("emblem-web") << QStringLiteral("applications-internet")));
-    ui_.action_New->setIcon(icons.getThemeIcon(QStringLiteral("document-new"), QStyle::SP_DesktopIcon));
+    ui_.action_AddURL->setIcon(icons.getThemeIcon(QStringLiteral("insert-link"), QStyle::SP_DesktopIcon));
```

Comment:
`insert-link` is a freedesktop.org key. DYK if this change still works on Windows?

Reply:
Thanks Charles, I really donn't know and was hoping @Pentaphon might be able to help there, as someone who uses Windows. MacOS and Manjaro are fine.

Reply:
Maybe I'm being too literal but I don't understand the answer. Does this change still work on Windows?

Reply:
It should, but can't confirm.